packages/mobile/src/components/Dialog.tsx

```diff
@@ -17,8 +17,8 @@ interface Props {
   image?: ImageSourcePropType
   title?: string | React.ReactNode
   children: React.ReactNode
-  actionText: string
-  actionPress: () => void
+  actionText?: string
+  actionPress?: () => void
```

Comment:
Well, I wasn't sure about this.
The primary action has some style that is different than the secondary action (green color text instead of gray).
In this case what we want is a Dialog that can be dismissed, and I think this will be a normal use case. That's why I allowed this behavior, but if you don't agree, what we can do is pass some style for the actionText.
However, I think it's great to have the possibility to use the Dialog in this case. Then it would be developer's responsibility to see how it's the best way to use the Dialog.

Reply:
ah I see, then maybe we could have add a `highlightPrimaryAction?: boolean`, which would default to `true`.
This way the highlighted style on the primary action would be configurable and we would ensure there's an action handler passed.
What do you think?

Reply:
Hmm... I'm not sure. What will be the expected behavior if `highlightPrimaryAction = false` and we also pass a secondaryAction?
I think it's ok to have the possibility to create a Dialog without any action. Why should we force it to have one?

I'm gonna try to merge this, but we can continue this discussion offline if you want (or we can keep using this thread), and I will create a follow up PR if needed, that sounds ok?

Reply:
Yes that's fine.